core/src/executor/evaluate/mod.rs

```diff
@@ -79,7 +79,7 @@ pub async fn evaluate<'a, T>(
                 0 => Err(EvaluateError::NestedSelectRowNotFound.into()),
                 1 => evaluations
                     .into_iter()
-                    .nth(0)
+                    .next()
                     .unwrap_or_else(|| Err(EvaluateError::NestedSelectRowNotFound.into())),
```

Comment:
Thanks for suggestion of take(2) way.
I tried take(2) and matching len().
But I'm not sure whether it's fine to use NestedSelectRowNotFound error again here🥲

Reply:
Good progress!
In this case, using `if` looks better than `match`

```rust
if evaluations.len() > 1 {
    return Err(EvaluateError::MoreThanOneRowReturned.into());
}

evaluations
    .into_iter()
    .next()
    .unwrap_or_else(|| Err(EvaluateError::NestedSelectRowNotFound.into()))
```